src/py/flwr/server/superlink/state/state_test.py

```diff
@@ -409,6 +414,91 @@ def test_num_task_res(self) -> None:
         # Assert
         assert num == 2
 
+    def test_server_public_private_key(self) -> None:
+        """Test store_server_public_private_key, get_server_private_key and
+        get_server_public_key from state."""
```

Comment:
First sentence of the docstring must fit on one line (same for some tests below)